plugin/src/main/kotlin/dev/icerock/gradle/talaiot/InfluxDbPublisher.kt

```diff
@@ -89,7 +89,9 @@ class InfluxDbPublisher(
     }
 
     private fun createTaskPoints(report: ExecutionReport): List<Point>? {
-        return report.tasks?.map { task ->
+        return report.tasks?.filter { task ->
+            task.ms.div(1000.0) >= 30.0
```

Comment:
we can use kotlin Duration api. 
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/
```
task.ms.milliseconds >= 30.seconds
```